src/components/cards/ShortCutWidgets.vue

```diff
@@ -0,0 +1,370 @@
+			@close="$emit('labelsassigned')">
+			<div class="modal__content labels">
+				<Multiselect ref="labelselect"
+					v-model="assignedLabels"
+					:style="{top:cardTop, left:cardLeft}"
+					:multiple="true"
+					:options="labelsSorted"
+					:placeholder="t('deck', 'Assign a tag to this card…')"
+					:taggable="true"
+					label="title"
+					track-by="id"
+					@select="addLabelToCard"
+					@remove="removeLabelFromCard">
+					<template #option="scope">
+						<div :style="{ backgroundColor: '#' + scope.option.color, color: textColor(scope.option.color)}" class="tag">
+							{{ scope.option.title }}
+						</div>
+					</template>
+					<template #tag="scope">
+						<div :style="{ backgroundColor: '#' + scope.option.color, color: textColor(scope.option.color)}" class="tag">
+							{{ scope.option.title }}
+						</div>
+					</template>
+				</Multiselect>
+			</div>
+		</Modal>
+		<Modal v-if="assignDueDate"
+			:title="card.title"
+			@close="$emit('duedatesassigned')">
+			<div class="modal__content duedate" :style="{top:cardTop, left:cardLeft}">
+				<Actions>
+					<ActionButton v-if="card.duedate" icon="icon-delete" @click="removeDue()">
+						{{ t('deck', 'Remove due date') }}
+					</ActionButton>
+				</Actions>
+				<DatetimePicker ref="dateselect"
+					v-model="duedate"
+					:placeholder="t('deck', 'Set a due date')"
+					type="datetime"
+					:minute-step="5"
+					:show-second="false"
+					:lang="lang"
+					:formatter="format"
+					:shortcuts="shortcuts"
+					confirm />
+			</div>
+		</Modal>
+	</div>
+</template>
+
+<script>
+import { mapState, mapGetters } from 'vuex'
+import moment from '@nextcloud/moment'
+import {
+	Actions,
+	ActionButton,
+	Avatar,
+	Modal,
+	Multiselect,
+	DatetimePicker,
+} from '@nextcloud/vue'
+import Color from '../../mixins/color'
+import {
+	getLocale,
+	getDayNamesMin,
+	getFirstDay,
+	getMonthNamesShort,
+} from '@nextcloud/l10n'
+
+export default {
+	name: 'ShortCutWidgets',
+	components: {
+		Actions,
+		ActionButton,
+		Avatar,
+		Modal,
+		Multiselect,
+		DatetimePicker,
+	},
+	mixins: [Color],
+	props: {
+		card: {
+			type: Object,
+			default: null,
+		},
+		assignUsers: {
+			type: Boolean,
+			default: false,
+		},
+		assignLabels: {
+			type: Boolean,
+			default: false,
+		},
+		assignDueDate: {
+			type: Boolean,
+			default: false,
+		},
+	},
+	data() {
+		return {
+			copiedCard: null,
+			assignedUsers: null,
+			assignedLabels: null,
+			locale: getLocale(),
+			lang: {
+				days: getDayNamesMin(),
+				months: getMonthNamesShort(),
+				formatLocale: {
+					firstDayOfWeek: getFirstDay() === 0 ? 7 : getFirstDay(),
+				},
+				placeholder: {
+					date: t('deck', 'Select Date'),
+				},
+			},
+			format: {
+				stringify: this.stringify,
+				parse: this.parse,
+			},
+			shortcuts: [
+				{
+					text: t('deck', 'Today'),
+					onClick() {
+						const date = new Date()
+						date.setDate(date.getDate())
+						date.setHours(23)
+						date.setMinutes(59)
```

Comment:
@luka-nextcloud No, because then a ticket is immediately marked as overdue even though you wanted it to be due the same day. Also consider usage across timezones. Therefore the timestamp has to be at the end of the day and not the current time.